Utilities/ReleaseScripts/scripts/duplicateReflexLibrarySearch.py

```diff
@@ -42,6 +42,7 @@
          {'L1TrackTrigger'        : ['(TTStub|TTCluster|TTTrack).*Phase2TrackerDigi']},
          {'L1TCalorimeterPhase2'  : ['l1tp2::CaloTower.*']},
          {'L1TCalorimeter'        : ['l1t::CaloTower.*']},
+         {'VertexFinder' : ['l1tVertexFinder::Vertex']},
```

Comment:
Could you format the line similarly to surrounding?
```suggestion
         {'VertexFinder'         : ['l1tVertexFinder::Vertex']},
```
(I hope I got the number of spaces correctly)